src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { validateRequest } = require('./convertToCase/validateRequest');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const { pathname, searchParams } = normalizedURL;
+
+    const originalText = pathname.slice(1);
+    const targetCase = searchParams.get('toCase');
+    const response = {
+      originalText,
+      targetCase,
+    };
```

Comment:
Move it to the separate variable
```suggestion
    const response = {
      originalText,
      targetCase,
    } = ....(...);
```